modules/xmpp/JingleSessionPC.js

```diff
@@ -637,6 +643,31 @@ export default class JingleSessionPC extends JingleSession {
     }
 
     /**
+     * Reads the "url" parameter in the <web-socket> tag of the jingle offer iq
+     * and stores it into this.bridgeWebSocketUrl.
+     * @param contets
+     */
+    readBridgeWebSocketUrl(contents) {
+        $(contents).each((i1, content) => {
+            if (this.bridgeWebSocketUrl) {
+                return;
+            }
+
+            const websockets
+                = $(content).find(
+                    'transport>web-socket');
+
+            websockets.each((i2, websocketElem) => {
```

Comment:
@bgrozev Do we ever expect to get more than one websocket elements? If not, you can skip the second iterator @ibc. Also, Boris, Iñaki is looking at the URL for any content, which is arguably wrong, but I don't see that ever breaking since we always one transport, do you think it's ok to leave it as-is?